crates/oxc_ast/src/ast/js.rs

```diff
@@ -419,7 +459,7 @@ impl<'a> Expression<'a> {
     }
 }
 
-/// Identifier Name
+/// Fundamental syntactic structure used for naming variables, functions, and properties. It must start with a Unicode letter (including $ and _) and can be followed by Unicode letters, digits, $, or _.: `var` in `let var = 1;`
```

Comment:
`cargo doc` shows the first line as the title, and the next lines as descriptions.

<img width="894" alt="image" src="https://github.com/oxc-project/oxc/assets/1430279/f252c4b8-63dc-4dfd-8e08-ef5f9736d5bf">

So the first line should be simple to understand 
<img width="617" alt="image" src="https://github.com/oxc-project/oxc/assets/1430279/1d896926-e6d8-40e4-9b6d-ef831ffe59b2">

Reply:
I see, understanding.